src/scenes/Activities/index.js

```diff
@@ -84,27 +120,36 @@ class Activities extends React.Component {
   }
 
   handleTabChange(event, value) {
-    this.setState({ tabIdx: value });
+    switch (value) {
+      case 0: {
```

Comment:
Can we use a const to store the tab index value?

Reply:
good idea 👍